lib/main.js

```diff
@@ -0,0 +1,4 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import './middleware.js';
```

Comment:
Maybe just move the contents of middleware into main.js for now?  Or if we expect lots of fixes, name them something like 'safari.js' or 'safari-samesite.js' so it's clear what the scope is?